src/main/groovy/com/mini/asaas/payment/PaymentRepository.groovy

```diff
@@ -25,7 +25,7 @@ class PaymentRepository implements Repository<Payment, PaymentRepository> {
     @Override
     List<String> listAllowedFilters() {
         return [
-                "customerId"
+                "customer.id"
```

Comment:
Para manter um padrão, como já foi feito no PayerRepository, melhor deixar aqui como:

```suggestion
                "customerId"
```

Reply:
Eu fiz dessa forma porque por algum motivo está gerando exceção. Eu até cheguei a olhar os domínios e entender o porquê, porém não consegui identificar.

`QueryException occurred when processing request: [GET] /payment/index
could not resolve property: customerId of: com.mini.asaas.payment.Payment.`

Reply:
No buildCriteria, dentro de addCriteria, você usou dessa forma aqui? 

```groovy
eq("customer.id", Long.valueOf(search.customerId.toString()))
```

Vi que você tinha feito passando eq("customerId"), daí dá esse erro mesmo.

Reply:
A ideia é só deixar o nome da chave do parâmetro de pesquisa como "customerId" para facilitar e padronizar, mas o primeiro argumento do método eq, tem que ser "customer.id"

Reply:
Ahh, não sabia, obrigada